### PR TITLE
[dimcli] Update to v7.2.0

### PR DESCRIPTION
**Pull request by gknowles**
If this PR updates an existing port, please uncomment and fill out this checklist:

- [ x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [ x] SHA512s are updated for each updated download.
- [ x] The "supports" clause reflects platforms that may be fixed by this new version.
- [ x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ x] Any patches that are no longer applied are deleted from the port's directory.
- [ x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [ x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/dimcli/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO gknowles/dimcli
-    REF v7.1.0
-    SHA512 3fad1c68e116c65df453a545fe9def6dc05941675900fd81728531d7638fc1814b82ee1f613ec451bc6c31702a3d5e31f275e605fe6bd62c61513ecd78a172cc
+    REF v7.2.0
```
Comment (WangWeiLin-MV) [inline ports/dimcli/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

Comment (gknowles) [reply ports/dimcli/portfile.cmake]: Not sure that it's an improvement, but if that's how things are normally done it seems fine.

File: versions/d-/dimcli.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "516af174c59826a049fb9bca271f58e9f1628dcc",
```
Comment (WangWeiLin-MV) [inline versions/d-/dimcli.json:4]: > The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.

Hi @gknowles, could you please help fix this hash?

Comment (gknowles) [reply versions/d-/dimcli.json]: Done

**Review by WangWeiLin-MV — APPROVED** [review]
Port test pass with following triplets:
* x64-linux
* x64-windows
* x64-windows-static
* x86-windows

**Pull request merged**